app.py

```diff
@@ -28,8 +28,8 @@
 
 torch.backends.cudnn.benchmark=True
 
-
-os.environ['CUDA_VISIBLE_DEVICES']='0'
+#uncomment below line to utilise Nvidea GPU's
+#os.environ['CUDA_VISIBLE_DEVICES']='0'
```

Comment:
going this path I would probably do it automagically;

In [1]: import torch

In [2]: torch.cuda.current_device()
Out[2]: 0

In [3]: torch.cuda.device(0)
Out[3]: <torch.cuda.device at 0x7efce0b03be0>

In [4]: torch.cuda.device_count()
Out[4]: 1

In [5]: torch.cuda.get_device_name(0)
Out[5]: 'GeForce GTX 950M'

In [6]: torch.cuda.is_available()
Out[6]: True